lisa/transformers/kernel_source_installer.py

```diff
@@ -395,6 +413,26 @@ def get_source_code(self) -> PurePath:
 
         return code_path
 
+    def get_details(self) -> Dict[str, Any]:
```

Comment:
It looks this information is not only for repo location. It should be moved up to source installer.